astro/src/content/docs/extend/events-and-webhooks/webhook-event-log.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Webhook Event Log
+description: Learn about FusionAuth's Webhook Event Log and how to use it to search and view details of past events.
+navcategory: developer
+section: extend
+subcategory: events and webhooks
+---
+import ApplicationWebhooksWarning from 'src/content/docs/extend/events-and-webhooks/_application-webhooks-warning.mdx';
+import Aside from 'src/components/Aside.astro';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import Icon from 'src/components/icon/Icon.astro';
+import InlineField from 'src/components/InlineField.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+import {RemoteCode} from '@fusionauth/astro-components';
+import DiagramTxNone from 'src/diagrams/docs/extend/events-and-webhooks/_transaction-none.astro';
+import DiagramTxAny from 'src/diagrams/docs/extend/events-and-webhooks/_transaction-any-succeed.astro';
+import DiagramTxThree from 'src/diagrams/docs/extend/events-and-webhooks/_transaction-any-succeed-three-failures.astro';
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
+<Aside type="version">
+Available since 1.53.0
+</Aside>
+
+## Overview
+
+A Webhook Event Log is provided to review events sent by FusionAuth along with timing and result information for each attempt to send the event to configured endpoints. The `webhook_event_log_viewer` role grants users access to these pages. You can find the Webhook Event Log by navigating to <Breadcrumb>System > Webhook Log</Breadcrumb>. The same information is available via the [Webhook Event Log APIs](/docs/apis/webhook-event-logs).
+
+## List of Webhook Event Logs
```

Comment:
I think that this header and all below it should be an H3, and get an additional H2 that says `Webhook Event Log UI`